configure.ac

```diff
@@ -9,7 +9,7 @@ AC_LANG([C++])
 PDNS_CHECK_OS
 # Check dependencies with pkgconfig
 PKG_PROG_PKG_CONFIG
-PKG_CHECK_MODULES([GEOIP], [geoip], [AC_DEFINE([HAVE_GEOIP], [1], [Define to 1 if you have geoip])], [true])
+PDNS_CHECK_GEOIP
```

Comment:
move this commit before the previous one?

Reply:
yeah good point